src/pages/Earnings.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <PrimaryLayout title="Earnings">
+    <section class='top-row'>
+      <span class='available-label'>Total Available HF: </span><span class='available-hf'>{{ availableHF.toLocaleString() }}</span>
+      <a href='' class='holofuel-link'><LeaveSiteIcon class='leave-site-icon' />Go to HoloFuel</a>
+    </section>
+    <section class='card row'>
+      <div class='column'>
+        <h3 class='card-title'>Weekly Earnings</h3>
+        <div class='hf-amount'>{{ weeklyEarnings.toLocaleString() }} HF</div>
+        <TmpGraphIcon class='graph' />
+      </div>
+      <div class='column main-links'>
+        <router-link to='/earnings/invoices' class='main-link'>
+          <PaymentIcon class='main-link-icon'/>Payments <RightChevronIcon class='right-chevron-icon' color="#606C8B" />
+        </router-link>
+        <router-link :to="{ path: '/earnings/invoices', query: { filter: 'unpaid'} }" class='main-link'>
+          <PaymentIcon class='main-link-icon'/>Unpaid & Late <RightChevronIcon class='right-chevron-icon' color="#606C8B" />
```

Comment:
Doesn't this need to be escaped?
```suggestion
          <PaymentIcon class='main-link-icon'/>Unpaid &amp; Late <RightChevronIcon class='right-chevron-icon' color="#606C8B" />
```